### PR TITLE
Add WWDCast by @sgl0v

### PR DESCRIPTION
**Pull request by sgl0v**
1. [X] Project URL: https://github.com/sgl0v/WWDCast
2. [X] Update contents.json instead of README 
3. [X] One project per pull request
4. [X] Avoid iOS or open-source in description as it is assumed
5. [X] Use this commit title format if applicable: Add app-name by @github-username
6. [X] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -13106,6 +13106,30 @@
...
+        "developer",
+        "rxswift"
+      ],
+      "description": "The unofficial WWDC iOS application to watch WWDC videos and sessions on your ChromeCast",
+      "tags": [
+        "swift",
+        "rxswift",
+        "mvvm"
+      ],
+      "license": "MIT",
```
Comment (vpeschenkov) [inline contents.json]: Could you replace "MIT" to "mit"?

File: contents.json [hunk 2]
```diff
@@ -13106,6 +13106,30 @@
...
+        "https://github.com/sgl0v/WWDCast/blob/master/.github/screen1.png",
+        "https://github.com/sgl0v/WWDCast/blob/master/.github/screen2.png",
+        "https://github.com/sgl0v/WWDCast/blob/master/.github/screen3.png"
+      ],
+      "category-ids": [
+        "education",
+        "developer",
+        "rxswift"
+      ],
+      "description": "The unofficial WWDC iOS application to watch WWDC videos and sessions on your ChromeCast",
```
Comment (vpeschenkov) [inline contents.json]: Could you remove "iOS" from the description?

**Comment by sgl0v** [comment]
@vpeschenkov Done.

**Comment by vpeschenkov** [comment]
@sgl0v sorry, I've missed the screenshots, they should be a direct link to an image. Could you replace those links to these:
https://raw.githubusercontent.com/sgl0v/WWDCast/master/.github/screen1.png
https://raw.githubusercontent.com/sgl0v/WWDCast/master/.github/screen2.png
https://raw.githubusercontent.com/sgl0v/WWDCast/master/.github/screen3.png
Thanks!

**Comment by vpeschenkov** [comment]
@sgl0v thank you!

**Pull request merged**